README.md

```diff
@@ -91,16 +91,16 @@ For other features,see the demos below:
 | feature        | status |                      quick start                      | desc                                                                 |
 | -------------- | :----: | :---------------------------------------------------: | -------------------------------------------------------------------- |
 | Go (TinyGo)    |   ✅   | [demo](https://mosn.io/layotto/#/en/start/wasm/start) | Compile Code written by TinyGo to \*.wasm and run in Layotto         |
-| Rust           |  TODO  |                         TODO                          | Compile Code written by Rust to \*.wasm and run in Layotto           |
-| AssemblyScript |  TODO  |                         TODO                          | Compile Code written by AssemblyScript to \*.wasm and run in Layotto |
+| Rust           |   ✅   | [demo](https://mosn.io/layotto/#/en/start/wasm/start) | Compile Code written by Rust to \*.wasm and run in Layotto           |
+| AssemblyScript |   ✅   | [demo](https://mosn.io/layotto/#/en/start/wasm/start) | Compile Code written by AssemblyScript to \*.wasm and run in Layotto |
```

Comment:
I think the demo links here are not specific enough, the same links can be taken out separately and it is recommended to put links to the directories of specific implementations in different languages.

For the following FaaS sections and other language versions the same is recommended.

Reply:
the doc are all missed now，i think it should open a new issue